.dockerignore

```diff
@@ -1,9 +1,16 @@
-.vagrant/
-_build/
-apps/aecuckoo/c_src
-apps/aecuckoo/priv/bin
-apps/aecuckoo/priv/lib
-apps/aecore/priv/keys
-system_test
-erl_crash.dump
-aevm_external
+*
+!rebar.config
+!rebar.lock
+!Makefile
+!rebar3
+!rebar.config.script
+!VERSION
+!apps
+!.git
+!otp_patches
```

Comment:
Some files in this folder are actually derived, that may defeat the whitelist.

See repo and https://github.com/aeternity/epoch/blob/d10630530df4254d925bd6784854f1fef358b429/Makefile#L280

Reply:
Does `make && docker build . -t aeternity/epoch:local && make && docker build . -t aeternity/epoch:local` build image only once? (Did not test.)